backend/src/relay.ts

```diff
@@ -60,11 +59,12 @@ export async function relayFromDownloadedArchive(
   chainName: ChainName,
   path: string,
   target: Target,
-  state: State,
+  lastProcessedBlock: number,
   signer: SignerWithAddress,
   batchBytesLimit: number,
   batchCountLimit: number,
 ): Promise<number> {
+  const polkadotAppsBaseUrl = polkadotAppsUrl(target.targetChainUrl);
```

Comment:
why not get the URL from config?

Reply:
Good question. I got lazy and decided to not add yet another argument to the function. It will have to be refactored later with some interfaces for such a big number of arguments.